### PR TITLE
TechSmithCorporation.Snagit12 version 12.4.1.3036

### PR DESCRIPTION
**Pull request by MEX-Kaz**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/91275)

### CODE REVIEW

File: manifests/t/TechSmithCorporation/Snagit12/12.4.1.3036/TechSmithCorporation.Snagit12.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: TechSmithCorporation.Snagit12
```
Comment (Trenly) [inline manifests/t/TechSmithCorporation/Snagit12/12.4.1.3036/TechSmithCorporation.Snagit12.installer.yaml:4]:
```suggestion
PackageIdentifier: TechSmith.Snagit.12
```

File: manifests/t/TechSmithCorporation/Snagit12/12.4.1.3036/TechSmithCorporation.Snagit12.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: TechSmithCorporation.Snagit12
```
Comment (Trenly) [inline manifests/t/TechSmithCorporation/Snagit12/12.4.1.3036/TechSmithCorporation.Snagit12.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: TechSmith.Snagit.12
```

File: manifests/t/TechSmithCorporation/Snagit12/12.4.1.3036/TechSmithCorporation.Snagit12.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: TechSmithCorporation.Snagit12
```
Comment (Trenly) [inline manifests/t/TechSmithCorporation/Snagit12/12.4.1.3036/TechSmithCorporation.Snagit12.yaml:4]:
```suggestion
PackageIdentifier: TechSmith.Snagit.12
```

File: manifests/t/TechSmithCorporation/Snagit12/12.4.1.3036/TechSmithCorporation.Snagit12.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: TechSmithCorporation.Snagit12
+PackageVersion: 12.4.1.3036
+PackageLocale: en-US
+Publisher: TechSmith Corporation
+PackageName: Snagit 12
+License: Licensed
```
Comment (Trenly) [inline manifests/t/TechSmithCorporation/Snagit12/12.4.1.3036/TechSmithCorporation.Snagit12.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

File: manifests/t/TechSmithCorporation/Snagit12/12.4.1.3036/TechSmithCorporation.Snagit12.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: TechSmithCorporation.Snagit12
+PackageVersion: 12.4.1.3036
+PackageLocale: en-US
+Publisher: TechSmith Corporation
+PackageName: Snagit 12
+License: Licensed
+Copyright: Copyright (c) TechSmith Corporation. All rights reserved.
+ShortDescription: Snagit 12
```
Comment (Trenly) [inline manifests/t/TechSmithCorporation/Snagit12/12.4.1.3036/TechSmithCorporation.Snagit12.locale.en-US.yaml:11]: Please provide a better short description

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by MEX-Kaz — COMMENTED** [review]
Updated short description

**Pull request closed without merging**